packages/migration/src/migrations/v0030_mark3pVideoAsMuted.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function mark3pVideoAsMuted({ pages, ...rest }) {
+  return {
+    pages: pages.map(reducePage),
+    ...rest,
+  };
+}
+
+function reducePage({ elements, ...rest }) {
+  return {
+    elements: elements.map(updateElement),
+    ...rest,
+  };
+}
+
+function updateElement(element) {
+  if (element.type === 'video' && element.resource?.id) {
+    const resourceId = element.resource.id.toString();
+    const isCoverrMedia = resourceId.startsWith('media/coverr');
+    if (isCoverrMedia) {
+      element.resource.isMuted = true;
+    }
```

Comment:
```suggestion
    element.resource.isMuted = isCoverrMedia;
```

Not sure if we should also set this to `false` if for all resources....

Reply:
I don't think so. The way it currently is is fine.